devdox/app/services/repository.py

```
@@ -1,56 +1,61 @@
 from typing import List, Tuple
 from uuid import UUID, uuid4
 from fastapi import Depends
-from tortoise.exceptions import IntegrityError
+from tortoise.exceptions import DoesNotExist, IntegrityError
```

Ruff (0.12.2)

`tortoise.exceptions.IntegrityError` imported but unused

Remove unused import: `tortoise.exceptions.IntegrityError`

(F401)